src/connector/src/source/reader/reader.rs

```diff
@@ -106,14 +106,9 @@ impl SourceReader {
     }
 
     /// Postgres and Oracle connectors need to commit the offset to upstream.
+    /// And we will spawn a separate tokio task to wait for epoch commit and commit the source offset.
```

Comment:
Didn't notice that we **spawn** a task. It would be better to keep the convention that an actor acts in a single-threaded manner, that is, can be concurrent but should not be parallel.

Reply:
I think you mean the compute logic of an actor? Here is pure control logic to let the actor aware the epoch commit event.